src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -20,15 +22,15 @@ class FormIntroButtons extends React.Component {
     let wasLoggedIn;
     let isLoggedIn;
     try {
-      wasLoggedIn = this.props.user.login.currentlyLoggedIn;
+      wasLoggedIn = this.props.loggedIn;
```

Comment:
This got overlooked in a refactor and I must not have tested it. That's why we write these things!